### ISSUE TITLE
Chest pain described as sharp or stabbing

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10137]
**Preferred term label:**
Chest pain described as sharp or stabbing

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0278145 though it is not chest specific

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Aortic dissection

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

This is [Sharp HP:0025281  ](https://hpo.jax.org/app/browse/term/HP:0025281)

I added stabbing as a synonym
